homeassistant/components/lookin/entity.py

```diff
@@ -61,7 +61,8 @@ class LookinDeviceCoordinatorEntity(
 
     def __init__(self, lookin_data: LookinData) -> None:
         """Init the lookin device entity."""
-        super().__init__(lookin_data.meteo_coordinator)
+        if lookin_data.meteo_coordinator:
+            super().__init__(lookin_data.meteo_coordinator)
```

Comment:
```suggestion
        assert lookin_data.meteo_coordinator is not None
        super().__init__(lookin_data.meteo_coordinator)
```

Reply:
It looks like this will never get created if there is no `meteo` sensor since there is now a guard in `sensor.py`

Reply:
yes, but mypy raise error here